Koans/AboutClasses.cs

```diff
@@ -0,0 +1,139 @@
+using System;
+using DotNetCoreKoans.Engine;
+using Xunit;
+
+namespace DotNetCoreKoans.Koans
+{
+    public class AboutClasses : Koan
+    {
```

Comment:
Can you give a little context here around what is to be learned and why it is important. This file might be a persons first introduction to OOP.

Reply:
I've tried to give a little introduction.